scalafix/output/src/main/scala/fix/Collectionstrawman_v0_TupleNZipped.scala

```diff
@@ -0,0 +1,20 @@
+package fix
+
+import strawman.collection.immutable.{ LazyList, List, Set }
+object Collectionstrawman_v0_Tuple2Zipped {
+  def zipped(xs: List[Int], ys: List[Int]): Unit = {
+    xs.lazyZip(ys)
+    coll(1).lazyZip(coll(2))
+    List(1, 2, 3).lazyZip(LazyList.from(1))
+  }
+  def coll(x: Int): List[Int] = ???
+}
+
+object Collectionstrawman_v0_Tuple3Zipped {
+  def zipped(xs: List[Int], ys: List[Int], zs: List[Int]): Unit = {
+    xs.lazyZip(ys).lazyZip(zs)
+    coll(1).lazyZip(coll(2)).lazyZip(coll(3))
+    List(1, 2, 3).lazyZip(Set(1, 2, 3)).lazyZip(LazyList.from(1))
+  }
```

Comment:
Apparently there isn't a consensus on the syntax for `lazyZip` in the discussions in #223. I chose `xs.lazyZip(ys).xxx(f)` over (xs lazyZip ys).xxx(f)`. Please let me know if you prefer the other option.

Reply:
I would go for `a.lazyZip(b)`, it's harder to get the parentheses right for infix operators. For example, `(a, (b, c)).zipped` becomes `a lazyZip ((b, c))` under -Yno-adapted-args. You might also have to add wrapping parentheses depending on the parent tree node `(a lazyZip b).c`